src/test/resources/com/puppycrawl/tools/checkstyle/checks/design/designforextension/InputDesignForExtension.java

```diff
@@ -0,0 +1,117 @@
+////////////////////////////////////////////////////////////////////////////////
+// Test case file for checkstyle.
+// Created: 2001
+////////////////////////////////////////////////////////////////////////////////
+package com.puppycrawl.tools.checkstyle.checks.design.designforextension;
+
+/**
+ * Test case for the "design for inheritance" check.
+ * @author Lars Kühne
+ **/
+public abstract class InputDesignForExtension
```

Comment:
@rnveach @romani  
It was used here under its original name here
[HideUtilityClassConstructorCheck](https://github.com/checkstyle/checkstyle/blob/34d0abd082756bdf9c864821b4225bc21922a0f0/src/test/java/com/puppycrawl/tools/checkstyle/checks/design/HideUtilityClassConstructorCheckTest.java#L88)
Before you merged it few hours earlier.
I've fixed this already.